identifiers/country-jp/multi_seat_constituencies.csv

```diff
@@ -0,0 +1,12 @@
+id,name
+ocd-division/country:jp/block_ed:chugoku,Chugoku

```

Comment:
What is the type `block_ed` inspired from? Since it is a multi-member constituencies, we can use `mmc`.

Reply:
Wikipedia uses "blocks". Changing this to `mmc`

Reply:
Aha, yes, they call them "proportional representation blocks": https://en.wikipedia.org/wiki/Tokyo_proportional_representation_block

I think just `block` is also fine. I just don't know that `block_ed` is quite right.